packages/eslint-config/index.js

```diff
@@ -13,6 +13,12 @@ module.exports = {
     jest: true,
   },
 
+  settings: {
+    'import/resolver': {
+      node: { paths: process.env.NODE_PATH.split(":") } // If ENV['NODE_PATH'] was provided we want to use it with eslint too
```

Comment:
are we sure that the `process.env.NODE_PATH` will exist to do a direct `split`? 🤔

Reply:
No, it's not guarenteed at all.  It's actually 100% optional.  But *if* it's included, then it will be used.  Otherwise, (I guess) NODE_PATH will be treated like an empty string which means `paths: []` - which is the current behavior.

Reply:
Hmm... actually, maybe my test was flawed...
```
root@e57b17a257a1:/app# node
> process.env.NODE_PATH.split(":")
[ '/var/cache/node_modules' ]
> process.env.NODE_PATHZZZ.split(":")
Thrown:
TypeError: Cannot read property 'split' of undefined
> 
```

Reply:
@samuelsilvadev - Okay I've fixed it and updated the PR.

Reply:
sounds great man @rkr-kununu 🤘